b2/console_tool.py

```diff
@@ -714,13 +719,21 @@ def run(self, args):
         max_workers = args.threads or 10
         self.api.set_thread_pool_size(max_workers)
 
+        #TODO pull out constants for min and max size, 100MB and 5GB
+        part_size_bytes = human2bytes(args.partSize) or human2bytes('100MB') #100MB default
+	if (part_size_bytes < human2bytes('100MB')) or (part_size_bytes > human2bytes('5GB')):
```

Comment:
This should use the minimum part size returned by the service, not the 100MB constant.